shared/src/common/enums/pagination/pagination-default-values.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum PaginationDefaultValues {
```

Comment:
```suggestion
enum PaginationDefaultValue {
```

enum's name should be singular 
don't forge about file name